src/layout/dom.ts

```diff
@@ -328,9 +350,6 @@ function metadata(tag: string, id: number, parentId: number): void {
                     }
                 }
                 break;
-            case "IFRAME":
-                if (config.lean === false) { value.metadata.boxmodel = true; }
```

Comment:
Is this the fix to not send iframes in box model?

Reply:
Yes, right.